src/uct/ib/mlx5/dc/dc_mlx5.h

```diff
@@ -154,6 +154,7 @@ typedef enum {
     UCT_DC_TX_POLICY_LAST
 } uct_dc_tx_policy_t;
 
+extern const char *uct_dc_tx_policy_names[];
```

Comment:
1. 2 space lines in H file
2. move it at line 353, next to uct_dc_mlx5_iface_config_table